src/test/java/com/ontotext/refine/cli/BaseProcessTest.java

```diff
@@ -56,11 +72,11 @@ void resetToOriginal() {
 
   @Test
   @ExpectSystemExit(ExitCode.USAGE)
-  void shouldFailOnMissingRefineUri() {
+  protected void shouldFailOnMissingRefineUrl() {
```

Comment:
I didn't saw override of this method so I think it can remain package private.

Reply:
It's overridden in MainTest to avoid the check for missing url.

Reply:
Oh, I've missed it.